app/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,7 @@
         android:label="@string/app_name"
         android:supportsRtl="false"
         android:theme="@style/AppTheme.DayNight.NoActionBar"
+        android:configChanges="orientation"
```

Comment:
You must include screenSize also here, read the `Caution: Beginning with Android 3.2 (...` here https://developer.android.com/guide/topics/resources/runtime-changes.html